src/ci/docker/host-x86_64/dist-x86_64-linux/build-clang.sh

```diff
@@ -26,8 +26,9 @@ hide_output \
       -DCMAKE_CXX_COMPILER=/rustroot/bin/g++ \
       -DCMAKE_BUILD_TYPE=Release \
       -DCMAKE_INSTALL_PREFIX=/rustroot \
+      -DCOMPILER_RT_BUILD_SANITIZERS=OFF \
       -DLLVM_TARGETS_TO_BUILD=X86 \
-      -DLLVM_ENABLE_PROJECTS="clang;lld" \
+      -DLLVM_ENABLE_PROJECTS="clang;lld;compiler-rt" \
```

Comment:
What's the rationale for these changes? They don't seem to be related to the rest of the PR, but I'm not familiar with clang build flags :)

Reply:
We need the profile runtime (from compiler-rt) to be able to use -fprofile-gen/use flags, so we need to build it here. Sanitizers are disabled because I couldn't get one of them to build (maybe due to our old gcc version or something, not sure, but doesn't seem worth debugging).